config/servicemappings/compute.yaml

```diff
@@ -499,6 +506,12 @@ spec:
                 version: v1beta1
                 group: compute.cnrm.cloud.google.com
               targetField: address
+            - key: addressIdRef
```

Comment:
This is going to trip up a lot of customers, I worry.  Can we write some code to make it better?

Reply:
Is it required to use selfLink of the ComputeAddress? If so, why?

Also, if this is required, I'd expect a more self-explanatory field name like `computeAddressSelfLinkRef` with proper field description clarifies that they must use this field when they use PSC?

Reply:
Yes, and I think that's due to the API design(while I'm unable to find an official field guide in their API doc, or in the TF doc). But in the TF doc, it uses the address.id in PSC forwarding rule example, address.address in the other examples. We should ask for a clear description for this exception from TF/API team.

I checked the http log, for PSC forwarding rule, the ip address field needs to be a full link like `"https://www.googleapis.com/compute/v1/projects/${projectId}/regions/us-central1/addresses/consumeraddress-${uniqueId}"`, using address(something like `10.0.0.2`) will cause the API error "The URL is malformed." 

What't the best practice to be able to reference to 2 fields in a resource, and minimize the impact to existing customers?

Reply:
I suggest we start with reusing the existing `addressRef` with its `external` field.